app/models/payment.rb

```diff
@@ -1,4 +1,4 @@
 class Payment < ApplicationRecord
-  validates_presence_of :credit_card_number, :credit_card_expiration_date, :result
+  validates_presence_of :credit_card_number, :result
```

Comment:
Removed credit card expiration date validation here because that data is coming in blank from the csv.